Userland/Libraries/LibGfx/Font/Typeface.cpp

```diff
@@ -61,8 +61,6 @@ void Typeface::set_vector_font(RefPtr<VectorFont> font)
 
 RefPtr<Font> Typeface::get_font(float point_size, Font::AllowInexactSizeMatch allow_inexact_size_match) const
 {
-    VERIFY(point_size > 0);
```

Comment:
:thonk: Do we want to verify that it's not negative though?

Reply:
:galaxythonk: I.. suppose!